### ISSUE TITLE
Unable to Subscribe and Delete RSS feeds

### ISSUE THREAD

**Issue by gunninks** [gpodder/mygpo#520]
I have noticed that I am not able to Subscribe and Delete RSS feeds. When clicking the link to subscribe or unsubscribe, it looks like its potentially doing something but nothing happens. Clearing cache and trying both firefox and chrome on Mac does not help.

**Comment by gunninks** [comment]
@elelay, yes, I am referring to gpodder.net. They link to this project. Should I have raise a bug somewhere else? Note this was linked from their website. Sorry in advanced if I did something wrong. Just trying to raise an issue. Thanks.

**Comment by elelay** [comment]
Don't worry: I transfered the bug to the right project. It's not obvious that gpodder.net = mygpo github project.
Hope you'll find an answer :-)

**Comment by aschoenr** [comment]
I'm having this issue as well.

**Comment by jeee** [comment]
same issue here

**Comment by peter-franken** [comment]
I have the same issue.

**Comment by aschoenr** [comment]
Issues exists in firefox and safari on Mac as well as firefox on android.

**Comment by gunninks** [comment]
Pardon my ignorance, what is the time frame to get this issue looked at?

**Comment by GiTimm** [comment]
I have the same issue.

**Comment by gunninks** [comment]
I've just stopped using it all together...

**Comment by mcepl** [comment]
There is just no way (via a device or via the web) to unsubscribe from a podcast so that it would stick. Hopefully, I will find somewhere else to sync my podcatcher with.

**Comment by stefangweichinger** [comment]
I also try to sync AntennaPod and the Garmin Podcasts App on my sport watch. I can't unsubscribe podcasts via webgui, seems to do nothing. Tried with Firefox under Fedora 34. Chrome doesn't even login.

**Comment by stefangweichinger** [comment]
It might be useless to add more details to the issue as there are >100 issues open here: doesn't seem to be actively maintained.

Maybe someone has a workaround or fix anyway.

**Comment by Ryochan7** [comment]
Seems like that is the case. There just needs to be a proper fork made. Not even sure there is a big enough demand for a service like this anymore.

**Comment by fundatillus** [comment]
It's been a while since this was logged and there have been many updates and maintenance done... 

Not sure what fixed this, but I had no problems subscribing and removing a feed today. Feel free to reopen if the problem comes up again.

**Comment by fundatillus** [comment]
Well now it's not working again... thanks for checking.

**Comment by Ryochan7** [comment]
The view calls a Celery task to actually perform the assigned function. I am still going with the assumption that Celery is not running on the server and that's why the functionality does not work. It works on my fork as I have used it myself to manage podcasts. Kind of pointless now though.

The alphagpo fork and alphagpo-feedservice code will remain on GitHub but I am sticking with the plan of shutting down the public instance at the end of July. AntennaPod planning to remove support for mygpo pretty much invalidates the need to use it.

**Comment by Ryochan7** [comment]
The forked public instance is no more. It has ceased to be.
